app/src/main/java/endava/codebase/android/movieapp/ui/component/ActorCard.kt

```diff
@@ -0,0 +1,71 @@
+package endava.codebase.android.movieapp.ui.component
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+
+data class ActorCardViewState(
+    val id: Int,
+    val imageUrl: String,
+    val name: String,
+    val character: String,
+)
+
+@Composable
+fun ActorCard(
+    actorCardViewState: ActorCardViewState,
+    modifier: Modifier = Modifier,
+) {
+    Card(
+        modifier = modifier,
+        backgroundColor = Color.White,
+        shape = RoundedCornerShape(15.dp),
+        elevation = 5.dp
+    )
+    {
+        Column(
+            modifier = Modifier.fillMaxSize()
+        ) {
+            AsyncImage(model = actorCardViewState.imageUrl, contentDescription = "Actor",
+                contentScale = ContentScale.Crop,
+                modifier = Modifier.fillMaxWidth().height(200.dp))
+
+            Text(
+                text = actorCardViewState.name,
+                fontSize = 20.sp,
+                maxLines = 2,
+                fontWeight = FontWeight.Bold,
+                modifier = Modifier.padding(all = 10.dp)
```

Comment:
If there is to big space between texts, you can just leave horizontal padding to `10.dp` and reduce vertical padding.